media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -3267,6 +3267,11 @@ do_exec2(struct mos_linux_bo *bo, int used, struct mos_linux_context *ctx,
         i915_execbuffer2_set_context_id(execbuf, ctx->ctx_id);
     execbuf.rsvd2 = 0;
 
+#ifdef ANDROID
+    i915_execbuffer2_set_tag(execbuf, tag);
```

Comment:
Here is a grep for the code:
```
[dvrogozh@dvrscl media-driver]$ fgrep -rsn i915_execbuffer2_set_tag
[dvrogozh@dvrscl media-driver]$
```

As you see i915_execbuffer2_set_tag is entirely missing. This code will not compile.